examples/rvc-app/rvc-common/rvc-app.matter

```diff
@@ -1370,6 +1660,26 @@ endpoint 1 {
     handle command OperationalCommandResponse;
     handle command GoHome;
   }
+
+  server cluster ServiceArea {
+    callback attribute supportedLocations;
+    callback attribute supportedMaps;
+    callback attribute selectedLocations;
+    callback attribute currentLocation;
+    callback attribute estimatedEndTime;
+    callback attribute progress;
+    callback attribute generatedCommandList;
```

Comment:
Not sure why these attributes that the generated code handles get marked callback rather than RAM. With the latest version of zap, there isn't even a choice for storage RAM/external